### DISCUSSION TITLE
Can pdfme fill existing form fields and ouput as filled info?

### DISCUSSION THREAD

**Discussion by allenhwkim** [pdfme/pdfme#89] [Q&A]
Hi,

I have pdf form template, and I want to fill the form fields and download the form as fields filled with values, e.g. firstName, lastName.
I searched several NodeJs solutions but not satisfactory. 
Can pdfme can do this task with nodejs?
If so, do you have any simple example?

Respectfully,

Allen Kim

**Comment by hand-dot** [comment]
@allenhwkim Hi!

Form of `pdfme/ui` is not compatible with the pdf form.
https://pdfme.com/docs/getting-started#form

Sory,

ref: https://github.com/pdfme/pdfme/issues/67
